### ISSUE TITLE
Dirac-trace identity not in Documentation

### ISSUE THREAD

**Issue by KevinB1995** [FormTracer/FormTracer#4]
The identity for doing the Dirac trace on a gamma5 matrix with four slashed vectors, which yields a Levi-Civita tensor, should definitely be included in the documentation, as it is easy to recognize in the attached example, however it might be very hard for a lot of people to recognize the e_[.,.,.,.] notation in a longer trace.

[Identity_Documentation_FORMTracer.pdf](https://github.com/FormTracer/FormTracer/files/3733187/Identity_Documentation_FORMTracer.pdf)

**Comment by vsht** [comment]
Just a comment of an outsider: The notation where Levi-Civita tensor has momenta instead of indices (if this is what your issue is about) is pretty much standard among all tools/codes for HEP calculations. FORM uses that as well and it is also mentioned somewhere in the manual. 

There is a very good technical reason to do so: It saves from the pain of canonicalizing dummy indices of an antisymmetric tensor, which is computationally expensive. The same goes for not distinguishing between upper and lower Lorentz indices.

**Comment by antoncyrol** [comment]
I don't think it makes sense to include this particular identity in the documentation because Form(-Tracer) uses many textbook identities. The shorthand notation is explained in section "2.2 Usage" of the paper. However, I think you've got a point here: e_[...] is the FORM notation for the epsilon tensor. The desired behaviour would be that it is translated back to your your notation eps[...]. I guess we simply forgot to add a replacement rule for this case, but I'm sure you can easily do it yourself.
